cmd/vic-machine/common/registries.go

```diff
@@ -40,7 +40,7 @@ func (r *Registries) Flags() []cli.Flag {
 	return []cli.Flag{
 		cli.StringSliceFlag{
 			Name:  "registry-ca, rc",
-			Usage: "Specify a list of additional certificate authority files to use to verify secure registry servers",
+			Usage: "Specify a list of certificate authority files to replace old ones to verify secure registry servers",
```

Comment:
For the first time, there is no any existing ones, so I suggest to remove "replace old ones" here.

Reply:
Done